pkg/block/metadata/deletionmark.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package metadata
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path"
+
+	"github.com/go-kit/kit/log"
+	"github.com/oklog/ulid"
+	"github.com/pkg/errors"
+	"github.com/thanos-io/thanos/pkg/objstore"
+	"github.com/thanos-io/thanos/pkg/runutil"
+)
+
+const (
+	// DeletionMarkFilename is the known json filename to store details about when block is marked for deletion.
+	DeletionMarkFilename = "deletion-mark.json"
+
+	// DeletionMarkVersion1 is the version of deletion-mark file supported by Thanos.
+	DeletionMarkVersion1 = 1
+)
+
+// ErrorDeletionMarkNotFound is the error when deletion-mark.json file is not found.
+var ErrorDeletionMarkNotFound = errors.New("deletion-mark.json not found")
+
+// ErrorUnmarshalDeletionMark is type wrapper for errors to indicate that deletion-mark.json file cannot be unmarshalled.
```

Comment:
It's not wrapper, it's just an error, right?

Reply:
Since I have changed this to just an error, this is not required now.